packages/material-ui/src/Step/Step.js

```diff
@@ -84,7 +85,7 @@ function Step(props) {
       {connector &&
         alternativeLabel &&
         !last &&
-        React.cloneElement(connector, { orientation, alternativeLabel })}
+        React.cloneElement(connector, { orientation, alternativeLabel, ...prevStepState })}
```

Comment:
I think you don't need to spread this. In this state it's inconsistent with the props of `Step`.

Reply:
are you sure? The step state is spread to the Connector in the Stepper Component (non-alternativeLabel).

Reply:
@jmaloon Nice catch. I was not looking the the alternativeLabel logic. Even more reason to document the props StepConnector uses. If we document the classes we should also document the props.